content/docs/user-guide/programming/memory/allocators.md

```diff
@@ -50,163 +48,34 @@ Each allocator commonly implements the `IAllocator` interface and uses a schema
 | AZ::PoolSchema |  A specialized schema that implements a small block allocator for managing small, high-throughput allocations. Objects are typically pooled at the cost of using more memory.  `PoolSchema` is not thread safe. If you need a thread-safe version, use `ThreadPoolSchema` or write custom code to handle the synchronization.   |
 | AZ::ThreadPoolSchema |  A thread-safe pool schema that uses thread local storage to implement a small block allocator for each thread.  Because the thread pool allocator creates separate pools for each thread, it uses somewhat more memory, especially for fixed pool sizes.   |
 
 ## Creating an Allocator 
 
-We recommend that each O3DE gem or logical subsystem create a `ChildAllocator` to properly tag the memory that it allocates. This practice makes it easier to budget resource usage and get a holistic view of it.
+We recommend that each O3DE gem or logical subsystem create a `ChildAllocatorSchema` to properly tag the memory that it allocates. This practice makes it easier to budget resource usage and get a holistic view of it.
+Tagged memory usage can be viewed or written to files in several ways including by running the `sys_DumpAllocators` or `sys_DumpAllocationRecordsToFile` console commands.
 
-If you choose to write your own schema, be aware that caching significant chunks of memory can be problematic. Such caching can hamper the ability of other systems to evolve to fit the content in your game. Unless you have specific requirements, we recommend that you create a `ChildAllocator` that eventually uses the `SystemAllocator`. Using a `ChildAllocator` ensures that your memory is as recoverable and reusable as possible.
+If you choose to write your own schema, be aware that caching significant chunks of memory can be problematic. Such caching can hamper the ability of other systems to evolve to fit the content in your game. Unless you have specific requirements, we recommend that you create a `ChildAllocatorSchema` that eventually uses the `SystemAllocator`. Using a `ChildAllocatorSchema` ensures that your memory is as recoverable and reusable as possible.
 
 **To create an allocator**
 
-1. Choose a schema to use, write a custom schema, or choose an existing allocator that you want to modify. For more information, see [AZ Allocator Schemas](#memory-allocators-az-allocator-schemas).
+1. Choose a schema to use, write a custom schema, or choose an existing allocator that you want to modify. For more information, see [AZ Allocator Schemas](#allocator-schemas).
 
 1. Inherit from `AllocatorBase`<*your\_schema*> to create your `Allocator` class.
 
-1. Add `AZ_TYPE_INFO` so that `AllocatorInstance<>` can properly manage your type.
+1. Add `AZ_RTTI` so that `AllocatorInstance<>` can properly manage your type.
```

Comment:
The numbering '1' is repeated? The above line too. Shouldnt this be 1, 2, 3... ?

Reply:
No because markdown will automatically increment the number when repeated lines of ".1" is used: https://www.seancdavis.com/posts/pro-tip-use-1-for-every-ordered-list-item-when-writing-markdown/

This is useful in avoid needing to update other lines in case an entry is added or removed.